lib/api/v3/utilities/path_helper.rb

```diff
@@ -130,6 +130,10 @@ def self.render_markup(format: nil, link: nil)
             path
           end
 
+          def self.show_revision(project_id, identifier)
+            show_revision_project_repository_path(project_id, identifier)
+          end
```

Comment:
Missing a spec ;-)

You can use the `#attachment_download` spec as orientation (as this one also points out of APIv3)